silent/tests/testdata/vu-group-semver-error.txt

```diff
@@ -2,7 +2,6 @@
 
 ! dependabot update -f input.yml --local . --updater-image ghcr.io/dependabot/dependabot-updater-silent
 # If this appears in the logs twice that means it tried to create a PR again
-stderr -count=1 DependencyFileNotParseable
```

Comment:
I'm handling this exception in the code that's why it won't appear in here anymore.

Reply:
You can switch this to check that stdout records the error now as dependency_file_not_parseable.

Reply:
The test is failing. It doesn't look like this exception is going into stdout